transcom/mymove
# INT-B-19918 do not send safety emails

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A947889&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary
> [!IMPORTANT]  
> Not testable until 18546 makes it into IntegrationTesting

Do not send emails for SAFETY moves

## Steps to test
1. Run the backend with EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run
2. Create a new customer and create a SAFETY move
3. Approve the move as a SC
4. Approve items as TOO
5. Make changes as Prime
6. Approve/reject as TIO
7. Make sure no emails were sent
